### PR TITLE
feat: Update styles

### PR DESCRIPTION
**Pull request by zinabat**
## Changes
- Moves styles to a separate `css` file
- Updates styles to be.. prettier (and look Pivotal-ish)

diagram_v2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ the blocker's descriptions. We use that to generate a
 an html page. This page will show basic details of the story when the
 corresponding node in the diagram is selected.
 
-![Example png diagram](./examples/diagram.png)
```
Comment (hockeybuggy) [inline README.md:16]: Would you mind deleting the v1 as well?

Comment (zinabat) [reply README.md:16]: Done

File: src/diagram_html_emitter.rs [hunk 2]
```diff
@@ -184,7 +144,7 @@ fn main(epic: &epic_info::Epic, stories: &Vec<epic_info::Story>) -> String {
         r#"
       <div class="panel">
         <div id="empty-state">
-         <p>Click a story to see it's details.</p>
+         <p>Click a story to see its details.</p>
```
Comment (hockeybuggy) [inline src/diagram_html_emitter.rs:147]: Good catch. :)

**Review by hockeybuggy — APPROVED** [review]
:thumbsup: Looks good to me :sparkles:

**Pull request merged**